packages/dm-core-plugins/src/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { TPlugin } from '@development-framework/dm-core'
+import Editor from './explorer/Editor'
+import { PluginComponent } from './yaml-view/YamlPlugin'
+import { TabsContainer } from './tabs/TabsContainer'
+import { MermaidComponent } from './mermaid-plugin/MermaidComponent'
+import { JobInputEdit } from './job/JobInputEdit'
+import { JobControlWrapper } from './job/JobControlWrapper'
+import { FormComponent } from './form/FormComonent'
+import Header from './header/Header'
+import { DefaultPreviewComponent } from './default-preview/DefaultPreviewComponent'
+import { DefaultPdfComponent } from './default-pdf/DefaultPdfComponent'
+import { EditBlueprint } from './blueprint/EditBlueprint'
```

Comment:
These names should perhaps be made more equal. Make a task for it?

Reply:
yes, there is some need for more consistent naming and structure for the plugins.
Creating new issue for this